import_flow_events.py

```diff
@@ -54,22 +59,63 @@
       utmTerm VARCHAR(40)
     );
 """
+Q_CREATE_METADATA_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_metadata (
+      flowId VARCHAR(64) NOT NULL UNIQUE SORTKEY,
+      beginTime TIMESTAMP NOT NULL,
```

Comment:
This is probably just me not understanding `DISTKEY` correctly yet. In my mental model, because `flowId` is unique here, it was the wrong choice. I thought the `DISTKEY` should be something for which there was more than one row of each. I'll go and RTFM again. 😄